packages/web3-core/src/index.js

```diff
@@ -22,11 +22,10 @@
 
 "use strict";
```

Comment:
These changes in a separate PR?

Reply:
I just changed the way it gets exported to be consistent with the other modules.